grpc/src/client/subchannel.rs

```diff
@@ -229,6 +235,7 @@ impl InternalSubchannel {
         transport: Arc<dyn Transport>,
         backoff: Arc<dyn Backoff>,
         unregister_fn: Box<dyn FnOnce(SubchannelKey) + Send + Sync>,
+        runtime: Arc<dyn Runtime>,
```

Comment:
Curious:

Given that we will have the same runtime for all the different gRPC components that require a runtime, did we consider something like a singleton that is initialized at init time, and all the components can use a getter to retrieve and use the singleton instead of the runtime being passed to every component that needs it?